netbox/core/views.py

```diff
@@ -715,6 +715,12 @@ def get(self, request, name):
         table = PluginVersionTable(plugin.release_recent_history, user=request.user)
         table.configure(request)
 
+        # If this is an HTMX request, return only the rendered table HTML
```

Comment:
Do we need to support a dynamic table here? It might be simpler to just render the static table without a paginator on the plugin page. It looks like `release_recent_history` is capped at five items anyway.